website/theme.config.tsx

```diff
@@ -33,6 +33,7 @@ export default defineConfig({
     return {
       description: description || `${siteName}: Modern API Platform and Ecosystem that scales`,
       openGraph: {
+        title: 'The Guild - GraphQL Tools',
```

Comment:
Why this change?

Reply:
<img width="1436" alt="Screenshot 2022-11-28 at 11 54 51" src="https://user-images.githubusercontent.com/37614975/204248104-dcbe32f2-91b8-4c7c-8e4f-5c798941895c.png">
To change the <meta property="og:title" content="Open Source – The Guild">
@B2o5T